save-core/src/commonTest/kotlin/org/cqfn/save/core/PerformAnalysisTest.kt

```diff
@@ -2,10 +2,12 @@ package org.cqfn.save.core
 
 import org.cqfn.save.core.config.ReportType
 import org.cqfn.save.core.config.SaveProperties
+import kotlin.test.Ignore
 import kotlin.test.Test
 
-class PluginDiscoveryTest {
+class PerformAnalysisTest {
     @Test
+    @Ignore
```

Comment:
I think it should work already, does it crash now?

Reply:
yeah, because oh not implemented reporter

Reply:
I've added a simple reporter in #113, later we can use some nice library to properly format a table

Reply:
removed ignore